apps/services/auth/api/src/app/modules/delegations/delegations.controller.spec.ts

```diff
@@ -286,6 +354,30 @@ describe('DelegationsController', () => {
                 ),
               ).toBeTruthy()
             })
+
+            it('should have updated delegation scopes field validTo to yesterday for deceased person link', async () => {
+              // Arrange
+              const delegationsModels = (
+                await delegationModel.findAll({
+                  include: [
+                    {
+                      model: DelegationScope,
+                      as: 'delegationScopes',
+                    },
+                  ],
+                })
+              ).map((delegation) => delegation.toDTO())
+
+              const allDelegationsScopes = delegationsModels
+                ?.map((delegation) => delegation?.scopes)
+                .filter(isDefined)
+                .flat()
+
+              // Assert
+              allDelegationsScopes?.forEach((scope) => {
+                expect(scope.validTo).toEqual(subDays(today, 1))
+              })
```

Comment:
Are there any delegation scopes in this array? Since this is testing deceased represented people and not custom delegations.

Reply:
This has now been updated to delete delegation instead.